plenum/test/view_change/test_view_change_wont_happen_if_ic_is_discarded.py

```diff
@@ -0,0 +1,57 @@
+import pytest as pytest
+
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+from plenum.test.test_node import ensureElectionsDone
+from plenum.test.view_change.helper import restart_node
+from stp_core.loop.eventually import eventually
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_interval = tconf.OUTDATED_INSTANCE_CHANGES_CHECK_INTERVAL
+    tconf.OUTDATED_INSTANCE_CHANGES_CHECK_INTERVAL = 5
+    yield tconf
+    tconf.OUTDATED_INSTANCE_CHANGES_CHECK_INTERVAL = old_interval
+
+
+def test_view_change_not_happen_if_ic_is_discarded(looper, txnPoolNodeSet,
+                                                   sdk_pool_handle,
+                                                   sdk_wallet_client,
+                                                   tconf, tdir, allPluginsPath):
+    """
+    1. panic_node (Delta) send InstanceChange for all nodes.
+    2. Restart nodes_to_restart (Beta, Gamma).
+    3. Wait OUTDATED_INSTANCE_CHANGES_CHECK_INTERVAL sec.
+    4. nodes_to_restart send InstanceChanges for all nodes.
+    5. Ensure elections done.
```

Comment:
Please mention that VC didn't happen since IC from the panic node was discarded by timeout.

Reply:
Done